code-challenges.test.js

```diff
@@ -10,51 +10,142 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+
+// Adding a describe method that lists the name of the function
+describe("fibonnaciSequence", () => {
+
+  // Adding a test method that describes what a function does
+  it("Takes in a number greater than 2 and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+
+    // Adding an expect method which calls on the fibonnaciSequence() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+    expect(fibonnaciSequence(6)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(fibonnaciSequence(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
+
+// Test fails with the ReferenceError: fibonnaciSequence is not defined
+
 // Example input: 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// b) Create the function that makes the test pass.
 
+// A Fibonacci Sequence is the series of numbers. Each number in the sequence is the sum of the two numbers that precede it. The sequence goes: 0, 1, 1, 2, 3, 5, 8, 13, 21, 34, etc.
 
-// b) Create the function that makes the test pass.
+// Creating a function with a parameter
+const fibSeq = (num) => {
 
+  // Adding an if statement that says "if num is greater than 2, return an array that's exactly that number of elements"
+  if (num > 2) {
 
+    // Creating a variable to hold the 2 numbers used to calculate the next number in the sequence
+    let myArray = [1,1]
+    
+    // Create a variable to store the new array
+    let newArray = []
 
-// --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
+    // Using the higher-order function .map() to iterate through the array, and return a new array of the same length.
+    // const newArray = myArray.map((value) => {
+    //   return value.length
+    // })
 
-// a) Create a test with expect statements for each of the variables provided.
+    // Actually, I'm going to use a for loop so I can see what's happening more clearly.
```

Comment:
This is a good move! everything here is top notch. WHen you are ready to do the logic (or math) it will go inside the { } of the for loop
```
for(i=0; i< whatever; i++) {
do logic here
}
```